### PR TITLE
[TTT] Setup playing a game of TTT

### PR DESCRIPTION
**Pull request by Alonski**
This PR sets up a game of TTT.
Currently, the game can be played between two players but doesn't handle:
* Validating moves: Player X can play 5 times in a row.
* Ending the game: Even when a player technically wins the game doesn't end.
* Challenging: Will be done later.
* Probably more things :)

Closes: #1119

### CODE REVIEW

File: packages/tic-tac-toe/app/controllers/game.ts [hunk 1]
```diff
@@ -28,19 +49,119 @@ export default class GameController extends Controller {
...
+        currentChannelState.channelId,
+        currentChannelState.aAddress,
+        currentChannelState.bAddress,
+        currentChannelState.aBal,
+        currentChannelState.bBal,
+        oPlayingState,
+        currentChannelState.aOutcomeAddress,
+        currentChannelState.bOutcomeAddress
+      );
+      console.log('Setting O on board');
```
Comment (spersico) [inline packages/tic-tac-toe/app/controllers/game.ts:146]: this looks really, really similar to [this](https://github.com/statechannels/monorepo/pull/1725/files#diff-1fb4bd5d34116c45523adad2892b02ecR107-R125). Couldn't it be refactored to a separate function that sets the state and updates the channel?

File: packages/tic-tac-toe/.env.development [hunk 2]
```diff
@@ -1,7 +1,7 @@
 DEFAULT_GAS=6721975
 DEFAULT_GAS_PRICE=20000000000
 GANACHE_HOST=127.0.0.1
-GANACHE_PORT=8547
+GANACHE_PORT=8545
```
Comment (spersico) [inline packages/tic-tac-toe/.env.development:4]: remember that env variables are now being handled globally with the `.env.ENVIRONMENT-USE-CASE-NAME-GOES-HERE` files, and `SC_ENV` flags before the run script

File: packages/tic-tac-toe/app/controllers/game.ts [hunk 3]
```diff
@@ -28,19 +49,119 @@ export default class GameController extends Controller {
...
+              this.Os = channelState.appData.Os;
+              console.log('Moving from xPlaying -> oPlaying');
+            }
+            if (ChannelState.inOPlaying(channelState)) {
+              this.player = 'x';
+              this.Xs = channelState.appData.Xs;
+              this.Os = channelState.appData.Os;
+              console.log('Moving from oPlaying -> xPlaying');
+            }
+          }
```
Comment (spersico) [inline packages/tic-tac-toe/app/controllers/game.ts:93]: same than my comment below. This looks, sort of, like repeated code

Comment (Alonski) [reply packages/tic-tac-toe/app/controllers/game.ts]: Done thanks

File: packages/tic-tac-toe/app/controllers/game.ts (hunk 1 above)
Comment (Alonski) [reply packages/tic-tac-toe/app/controllers/game.ts]: Done thanks!

File: packages/tic-tac-toe/.env.development (hunk 2 above)
Comment (Alonski) [reply packages/tic-tac-toe/.env.development:4]: I don't really understand this. Can you explain? Either here or in Slack?

**Pull request merged**